import_export/formats/base_formats.py

```diff
@@ -230,6 +230,9 @@ def create_dataset(self, in_stream):
             dataset.append(row_values)
         return dataset
 
+    def export_data(self, dataset, **kwargs):
+        return self.get_format().export_set(dataset, **kwargs)
+
```

Comment:
Is this necessary? It looks like it does not change what would parent method do.